apps/backend/app/api/routes/transformations.py

```
@@ -1511,4 +1522,372 @@ async def clear_history(
+    return BulkTransformationService()
+
+
+@router.post(
+    "/datasets/{dataset_id}/columns/select-pattern",
+    response_model=ColumnSelectionResponse
+)
+async def select_columns_by_pattern(
+    dataset_id: str,
+    request: ColumnSelectionPatternRequest,
+    current_user_id: str = Depends(get_current_user_id),
+    bulk_service: "BulkTransformationService" = Depends(get_bulk_transformation_service)
```

Ruff (0.14.10)

Undefined name `BulkTransformationService`

(F821)